styles/globalStyles.jsx

```diff
@@ -2,10 +2,15 @@ import { createGlobalStyle } from "styled-components";
 
 const GlobalStyles = createGlobalStyle`
 :root {
+  // Colors //
```

Comment:
to // comments work in styled template strings? I am not sure actually.

Reply:
It works on my machine :)